test/src/tests-batchUploader.ts

```diff
@@ -305,7 +424,11 @@ describe('batch uploader', () => {
             // start.
 
             window.mParticle._resetForTests(MPConfig);
-            
+
+            // TODO: For some reason, with offline batching enabled,
```

Comment:
is this outdated now? i believe only the batchuploader for dom and window events are failing now right?